auth-server/src/services/email.ts

```diff
@@ -76,5 +76,36 @@ export const sendResetPasswordEmail = (user: User, token: PasswordResetToken) =>
 	};
 
 	sgMail.send(msg).catch(e =>
-		console.error('Password reset email not sent',e));
+		console.error('Password reset email not sent', e));
+};
+
+export const sendPostSubscriptionMail = (user: User, author: User, comment) => {
+	if (!apiKey) {
+		console.warn('There is a new comment on the post you are subscribed to');
+		return;
+	}
+
+	const text = `
+		<p>
+			Hi ${user.name || ''}!<br/><br/>
+
+			<br />
+			${author.username} has commented on <a href="${DOMAIN}/post/${comment.id}">post</a>.<br /><br />
```

Comment:
A link to a comment.id doesn't bring anywhere as of now unfortunately. we should link to the `comment.post_id`
```suggestion
			${author.username} has commented on a <a href="${DOMAIN}/post/${comment.post_id}">post you subscribed to</a>.<br /><br />
```

Reply:
ah sorry